### ISSUE TITLE
Error during Opening the url of edit document

### ISSUE THREAD

**Issue by ahmedaswai** [petrsvihlik/WopiHost#28]
I am getting this error when I was trying to open the edit url 
 by the next url 

http://owas.cmsad.local/we/wordeditorframe.aspx?WOPISrc=http%3a%2f%2fowas.cmsad.local%2fhosting%2fdocs%2ftest.docx&access_token=IXuJCdjmmhc%3dHfPAmKzzT%2bs7zJpT4czLS4SVv1kBeGsIpbtJxaq%2f6mQ%3d&ui=zh-CN

> Exception	WAC Server FrontEnd unhandled exception [0] Microsoft.Office.Web.Common.EnvironmentAdapters.FileUnknownException: required URL paramters are not present     at Microsoft.Office.Web.Apps.Common.WopiDocument.CreateFromQuerySignature(String querySignature)     at Microsoft.Office.Web.Apps.Environment.WacServer.WSMainStorage.GetBaseDocumentFromContext(HttpContext context, String rawHostDocumentId)     at Microsoft.Office.Web.Host.WacServer.WebOneNote.WoncaFrame.OnPreRender(EventArgs e)     at Microsoft.Office.Web.Host.WacServer.WebOneNote.WordEditorFrame.OnPreRender(EventArgs e)     at System.Web.UI.Control.PreRenderRecursiveInternal()     at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint) StackTrace:  at uls.native.dll: (sig=2...	6b5565bf-0574-4f6e-8c6c-f9d0c1549913

**Comment by petrsvihlik** [comment]
afaik, the only required parameters are WOPISrc and access_token so it should not throw this kind of exception. try removing some characters from the parameter values to see whether some of them is causing the exception.

**Comment by ahmedaswai** [comment]
I removed characters from the access_token and from the WOPISrc and I am still getting the same exception 

I am using OfficeWebapps Preview 2016 

Exception	WAC Server FrontEnd unhandled exception [0] Microsoft.Office.Web.Common.EnvironmentAdapters.FileUnknownException: required URL paramters are not present     at Microsoft.Office.Web.Apps.Common.WopiDocument.CreateFromQuerySignature(String querySignature)     at Microsoft.Office.Web.Apps.Environment.WacServer.WSMainStorage.GetBaseDocumentFromContext(HttpContext context, String rawHostDocumentId)     at Microsoft.Office.Web.Host.WacServer.WebOneNote.WoncaFrame.OnPreRender(EventArgs e)     at Microsoft.Office.Web.Host.WacServer.WebOneNote.WordEditorFrame.OnPreRender(EventArgs e)     at System.Web.UI.Control.PreRenderRecursiveInternal()     at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint) StackTrace:  at uls.native.dll: (sig=2...	6017030a-ab6c-45c4-a8d3-20f3067b545d

**Comment by petrsvihlik** [comment]
I haven't tested it with the new preview yet. I'll probably do that sometime next week and let you know.

**Comment by ahmedaswai** [comment]
Your help is totally appreciated

**Comment by petrsvihlik** [comment]
Hi, I just tested it with Office Online Server 2016 RTM and it works fine. There were some issues earlier which I have fixed so make sure you pull the latest version.

Where did you read the exception above from? The event viewer?

If you still get an exception, make sure you also check the log files:
https://blogs.msdn.microsoft.com/zwsong/2013/04/16/where-would-you-find-uls-logs-on-office-web-apps-owa-2013-server/
